index.markdown

```diff
@@ -482,6 +482,18 @@ assumptions about your attendees. Get someone who is demographically very
 different from you to check your marketing material through to see if it makes
 sense and isn't offensive to someone who doesn't share your background.
 
+Many event attendees expect events to have and enforce a code of conduct. Having
+such a code encourages diversity of participation and creates and maintains a
```

Comment:
minor nit - this wording implies that you can't have a welcoming atmosphere without a CoC. I would say "can help to create and maintain"